shared/formatConversionTestUtil.ts

```diff
@@ -1,3 +1,5 @@
+// TODO: remove duplicate file
```

Comment:
@matthewlipski this file now lives in 3 spots. If the code is exactly the same - I think best to import the one from @shared?